public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js

```diff
@@ -127,10 +127,6 @@ jQuery.Class("OSSTimeControl_Calendar_Js", {
 							(event.subprocl ? '<div><span class="userIcon-' + event.subprocm + '" aria-hidden="true"></span> <label>' + app.vtranslate('JS_SUB_PROCESS') + '</label>: <a target="_blank" href="index.php?module=' + event.subprocm + '&view=Detail&record=' + event.subprocess + '">' + event.subprocl + '</a></div>' : '') +
 							(event.smownerid ? '<div><span class="glyphicon glyphicon-user" aria-hidden="true"></span> <label>' + app.vtranslate('JS_ASSIGNED_TO') + '</label>: ' + event.smownerid + '</div>' : '')
 				});
-				element.find('.fc-content, .fc-info').click(function () {
```

Comment:
why it has been removed

Reply:
This event needed changes in the library to work. I exchange it with the bulit-in library "eventClick".

Reply:
This event was needed becose of our changes in the library. Now clicked record automatically redirects.